src/components/IOUConfirmationList.js

```diff
@@ -258,6 +328,54 @@ class IOUConfirmationList extends Component {
         ];
     }
 
+    /**
+     * Gets the user's phone number from their secondary login.
+     * Returns null if it doesn't exist.
+     * @param {Array<Object>} loginList
+     *
+     * @returns {String|null}
+     */
+    getPhoneNumber(loginList) {
+        const secondaryLogin = _.find(loginList, login => Str.isSMSLogin(login.partnerUserID));
+        return secondaryLogin ? Str.removeSMSDomain(secondaryLogin.partnerUserID) : null;
+    }
+
+    /**
+     * Adds Venmo, if available, as the second option in the menu of payment options
+     */
+    addVenmoPaymentOption() {
+        // Add Venmo option
+        if (this.props.localCurrencyCode === CONST.CURRENCY.USD && this.isValidUSPhone(this.userPhoneNumber)) {
+            isAppInstalled('venmo')
+                .then((isVenmoInstalled) => {
+                    // We will return early if the component has unmounted before the async call resolves. This prevents
+                    // setting state on unmounted components which prints noisy warnings in the console.
+                    if (!isVenmoInstalled || !this.isComponentMounted) {
+                        return;
+                    }
+
+                    this.setState(prevState => ({
+                        confirmationButtonOptions: [...prevState.confirmationButtonOptions.slice(0, 1),
+                            {text: this.props.translate('iou.settleVenmo'), icon: Venmo},
+                            ...prevState.confirmationButtonOptions.slice(1),
+                        ],
+                    }));
+                });
+        }
+    }
+
+    /**
+     * @param {String} phoneNumber
+     * @returns {Boolean}
+     */
+    isValidUSPhone(phoneNumber) {
+        // Remove alphanumeric characters and validate that this is in fact a phone number
+        return CONST.REGEX.PHONE_E164_PLUS.test(phoneNumber.replace(CONST.REGEX.NON_ALPHA_NUMERIC, ''))
+
+            // Next make sure it's a US phone number
+            && CONST.REGEX.US_PHONE.test(phoneNumber);
```

Comment:
NAB: The max char limit on line has been increased, maybe we could put the entire expression on one line, this seems little bit weird with the other comment too 😅